### ISSUE TITLE
running droppy start

### ISSUE THREAD

**Issue by dtuando** [droppyjs/droppy#45]
im having trouble getting this to work. Ive tried both the install script and following the instructions from the Documentation. I run into 

```
Missing devDependencies to compile resource cache, please reinstall or run `npm install --only=dev` inside the project directory
    at compile (/usr/lib/node_modules/@droppyjs/cli/node_modules/@droppyjs/server/lib/services/resources.js:261:15)
    at ReadFileContext.callback (/usr/lib/node_modules/@droppyjs/cli/node_modules/@droppyjs/server/lib/services/resources.js:181:14)
    at FSReqCallback.readFileAfterOpen [as oncomplete] (fs.js:281:13)
```

**Comment by markhughes** [comment]
Hey @dtuando are you using `yarn` for this?

**Comment by dtuando** [comment]
Yes. Using yarn

**Comment by Tronic** [comment]
I've still had installation problems with the latest release too, looks like that needs more work. I can help debugging this if need to be, although I cannot currently remember *how* it is broken or what my workaround to get it running/upgraded was.

**Comment by markhughes** [comment]
@dtuando can you drop your version of node and npm please?

**Comment by DerTobi75** [comment]
Got the same error after installing with npm on Ubuntu 20.04.4 LTS:

```bash
tobias@hpserver:~$ npm install @droppyjs/cli
npm WARN deprecated fsevents@2.1.3: "Please update to latest v2.3 or v2.2"

> core-js@3.21.1 postinstall /home/tobias/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js:
> https://opencollective.com/core-js 
> https://patreon.com/zloirock 
> https://paypal.me/zloirock 
> bitcoin: bc1qlea7544qtsmj2rayg0lthvza9fau63ux0fstcz 

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)

npm WARN saveError ENOENT: no such file or directory, open '/home/tobias/package.json'
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN enoent ENOENT: no such file or directory, open '/home/tobias/package.json'
npm WARN pdfjs-dist@2.9.359 requires a peer of worker-loader@^3.0.7 but none is installed. You must install peer dependencies yourself.
npm WARN tobias No description
npm WARN tobias No repository field.
npm WARN tobias No README data
npm WARN tobias No license field.

+ @droppyjs/cli@0.2.2
added 118 packages from 1059 contributors and audited 119 packages in 14.102s

16 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```
npm --version: 6.14.16
node --version: v12.22.10

**Comment by markhughes** [comment]
@DerTobi75 you should install the cli globally 
npm install-g  @droppyjs/cli

**Comment by DerTobi75** [comment]
@markhughes 
That results in the same Errors. I tried to install without -g because someone mentions that fixed the problem for him.

**Comment by DerTobi75** [comment]
@markhughes 

This is what I got, when I try to install the cli globally:

```bash
sudo npm install -g @droppyjs/cli
[sudo] password for tobias:
npm WARN deprecated fsevents@2.1.3: "Please update to latest v2.3 or v2.2"
/usr/bin/droppy -> /usr/lib/node_modules/@droppyjs/cli/bin/droppy

> core-js@3.21.1 postinstall /usr/lib/node_modules/@droppyjs/cli/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js:
> https://opencollective.com/core-js
> https://patreon.com/zloirock
> https://paypal.me/zloirock
> bitcoin: bc1qlea7544qtsmj2rayg0lthvza9fau63ux0fstcz

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/@droppyjs/cli/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN pdfjs-dist@2.9.359 requires a peer of worker-loader@^3.0.7 but none is installed. You must install peer dependencies yourself.

+ @droppyjs/cli@0.2.2
added 118 packages from 1059 contributors in 11.11s
```


```bash
tobias@hpserver:~$ droppy start -c /home/tobias/.droppy/config -f /home/tobias/.droppy/files

           .:.
    :::  .:::::.   @droppyjs/server 0.2.2 running on node 12.22.10
  ..:::..  :::     config at /home/tobias/.droppy/config
   ':::'   :::     files at /home/tobias/.droppy/files
     '

2022-02-25 00:17:50 [INFO] Configuration: { listeners: [ { host: [ '0.0.0.0', '::' ], port: 8989, protocol: 'http' } ], public: false, timestamps: true, linkLength: 5, linkExtensions: false, logLevel: 2, maxFileSize: 0, updateInterval: 1000, pollingInterval: 0, keepAlive: 20000, uploadTimeout: 604800000, allowFrame: false, readOnly: false, ignorePatterns: [], watch: true, headers: {}}
2022-02-25 00:17:50 [INFO] Loading resources ...
2022-02-25 00:17:50 [INFO] ENOENT /home/tobias/.droppy/cache/cache.json, building cache ...
2022-02-25 00:17:50 [INFO] Loading resources done
2022-02-25 00:17:50 [ERROR] Missing devDependencies to compile resource cache, please reinstall or run `npm install --only=dev` inside the project directory
    at compile (/usr/lib/node_modules/@droppyjs/cli/node_modules/@droppyjs/server/lib/services/resources.js:261:15)
    at ReadFileContext.callback (/usr/lib/node_modules/@droppyjs/cli/node_modules/@droppyjs/server/lib/services/resources.js:181:14)
    at FSReqCallback.readFileAfterOpen [as oncomplete] (fs.js:261:13)
```

**Comment by markhughes** [comment]
Sorry all. I see the issue, my mistake - fix is incoming.

**Comment by markhughes** [comment]
Resolved in 0.2.3
